src/puppetlabs/trapperkeeper/bootstrap.clj

```diff
@@ -5,38 +5,40 @@
             [clojure.java.io :as io]
             [clojure.tools.logging :as log]
             [puppetlabs.trapperkeeper.internal :as internal]
-            [puppetlabs.trapperkeeper.services :as services]))
+            [puppetlabs.trapperkeeper.services :as services]
+            [puppetlabs.trapperkeeper.common :as common]
+            [schema.core :as schema]
+            [me.raynes.fs :as fs]))
 
+;; Schemas
+(def BootstrapLine {:namespace schema/Str :service-name schema/Str})
+(def BootstrapFiles [(schema/protocol io/IOFactory)])
+
+;; Constants
 (def bootstrap-config-file-name "bootstrap.cfg")
 
-(defn- parse-bootstrap-line!
+(schema/defn ^:private parse-bootstrap-line! :- BootstrapLine
```

Comment:
Not sure if it's better to mark these `^:private` or just create a `private` "pragma" section for these, like:

````clj
;;;;;;;;;;;;;;;;;;;;;;;
;;; Private
````
@KevinCorcoran - do you know if we have a standard / preference for what we're doing on these now?